dashboard/app/views/levels/_level_group.haml

```diff
@@ -51,6 +51,8 @@
         = render partial: 'levels/single_text_match', locals: {standalone: false, level: level, last_attempt: last_attempt }
       - elsif level_class == "free_response"
         = render partial: 'levels/free_response', locals: {in_level_group: true, level: level, last_attempt: last_attempt}
+      - else
+        = raise "Unsupported level type #{level_class}"
```

Comment:
True. I put this in just to help me debug something, if its not useful then I'll take it out